plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestDefaultJdbcMetadata.java

```diff
@@ -118,24 +115,24 @@ public void tearDown()
     @Test
     public void testListSchemaNames()
     {
-        assertTrue(metadata.listSchemaNames(SESSION).containsAll(ImmutableSet.of("example", "tpch")));
+        assertThat(metadata.listSchemaNames(SESSION).containsAll(ImmutableSet.of("example", "tpch"))).isTrue();
```

Comment:
I guess things like this could be improved further (`assertThat(...).containsAll(...)`?)

Reply:
> I applied more rules :) Is it better now?

Yes :)